x-pack/plugins/fleet/public/components/transform_install_as_current_user_callout.tsx

```diff
@@ -15,7 +15,10 @@ import type { PackageInfo } from '../../common';
 export const getNumTransformAssets = (assets?: PackageInfo['assets']) => {
   if (
     !assets ||
-    !(Array.isArray(assets.elasticsearch?.transform) && assets.elasticsearch?.transform?.length > 0)
+    !(
+      Array.isArray(assets.elasticsearch?.transform) &&
+      (assets.elasticsearch?.transform?.length ?? 0) > 0
```

Comment:
What's the reason for this change? small bug in the UI?

Reply:
made `transform` optional in TS type as it came up as an error in schema validation (sometimes undefined), and then tsc started complaining about `transform?.length` possibly undefined